src/main/java/com/volmit/iris/Iris.java

```diff
@@ -267,8 +268,8 @@ public void onDisable() {
             proj.close();
 
             for (World i : Bukkit.getWorlds()) {
```

Comment:
This is unrelated to Pregen, make seperate PRs on SEPERATE BRANCHES on your repo, otherwise, two PRs trying to merge the same branch at different times will still have the newest changes.

Reply:
yup